### PR TITLE
Use more varied set of todo text rather than `Something to do ${i}`

### PR DESCRIPTION
**Pull request by flashdesignory**
chrome before:
before_chrome

chrome after:
after_chrome

safari before:
before_apple

safari after:
after_safari

firefox before:
before_firefox

firefox after:
after_firefox

@kara

### CODE REVIEW

File: resources/translations.mjs [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        " Intelligent Fresh Mouse",
+        " Tasty Granite Gloves",
+        " Small Steel Shirt",
+        " Refined Concrete Computer",
+        " Sleek Frozen Shirt",
+        " Intelligent Concrete Shoes",
+        " Handmade Rubber Car",
+        " Sleek Cotton Towels",
+        " Unbranded Concrete Hat",
+        " Incredible Plastic Fish",
```
Comment (bgrins) [inline resources/translations.mjs:57]: Is the space in front of each string intentional?

Comment (rniwa) [reply resources/translations.mjs:57]: Yeah, these leading whitespace seems unnecessary.

Comment (flashdesignory) [reply resources/translations.mjs]: removed!

**Review by bgrins — APPROVED** [review]
I think this PR can be renamed to something like "Use more varied set of todo text rather than `Something to do ${i}`" (i.e. not refer to this as translating as-is), but I think the idea makes sense

**Review by rniwa — APPROVED** [review]

**Pull request merged**